source/crud/tests/v2/aggregate-merge.yml

```diff
@@ -0,0 +1,136 @@
+data:
+    - { _id: 1, x: 11 }
+    - { _id: 2, x: 22 }
+    - { _id: 3, x: 33 }
+
+minServerVersion: '4.1.12'
```

Comment:
I'm not sure exactly when `$merge` was introduced, but many of the SERVER tickets in its epic have this fixVersion so this seems safe to use. We're also just testing basic usage of `$merge`, so it's not a problem if its more complex options land in a later version.

Reply:
I would just set this to 4.2.0 since rc0 has been released

Reply:
Done and pushed. Resolving.